ui/src/pages/clearing/ClearingOfferModal.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import ModalFormErrorHandled from '../../components/Form/ModalFormErrorHandled';
+import { ServicePageProps, damlSetValues, makeDamlSet } from '../common';
+import { useStreamQueries } from '../../Main';
+import { Offer } from '@daml.js/da-marketplace/lib/Marketplace/Clearing/Service';
+import {
+  Service,
+  RequestOpenAccount,
+  RequestOpenAllocationAccount,
+} from '@daml.js/da-marketplace/lib/Marketplace/Custody/Service';
+import { CreateEvent } from '@daml/ledger';
+import { DropdownItemProps, Form, Button } from 'semantic-ui-react';
+import { AssetSettlementRule } from '@daml.js/da-marketplace/lib/DA/Finance/Asset/Settlement';
+import { AllocationAccountRule } from '@daml.js/da-marketplace/lib/Marketplace/Rule/AllocationAccount/module';
+import { createDropdownProp } from '../common';
+import { useParty, useLedger } from '@daml/react';
+import { Service as CustodyService } from '@daml.js/da-marketplace/lib/Marketplace/Custody/Service/';
+import { usePartyName } from '../../config';
+import {
+  OpenAccountRequest,
+  OpenAllocationAccountRequest,
+} from '@daml.js/da-marketplace/lib/Marketplace/Custody/Model';
+import { VerifiedIdentity } from '@daml.js/da-marketplace/lib/Marketplace/Regulator/Model';
+
+type OfferProps = {
+  offer?: CreateEvent<Offer>;
+};
+
+const ClearingOfferModal: React.FC<ServicePageProps<Service> & OfferProps> = ({
```

Comment:
I'd like to add this functionality (the auto-requesting) to the notifications/request dialogs as well, but for now it will only show up when you accept a clearing offer from the Clearing Network page.